core/src/main/scala/org/apache/spark/sql/delta/schema/SchemaUtils.scala

```diff
@@ -190,6 +191,13 @@ object SchemaUtils {
     if (dataFields.subsetOf(tableFields)) {
       data.toDF()
     } else {
+      // Allow the same shortcut logic if the only extra fields are CDC metadata fields.
```

Comment:
what does "same shortcut logic" mean? same as what?

Reply:
the above `if` statement. I have updated this comment.